semantic-release/action.yml

```
@@ -39,6 +46,8 @@ runs:
           ["main", "master"]
       env:
         GITHUB_TOKEN: ${{ inputs.github_token }}
+        NPM_TOKEN: ${{ inputs.npm_token }}
+        NODE_AUTH_TOKEN: ${{ inputs.node_auth_token }}
```

Are they needed?
Could we not use:
```suggestion
        NPM_TOKEN: ${{ inputs.github_token }}
        NODE_AUTH_TOKEN: ${{ inputs.github_token }}
```

---

@oskarkivra wouldn't that change make other packages get published on npm too? 😬

---

How does one package config know that it should be published as npm package? Is it https://github.com/kivra/react-components/blob/main/package.json#L4-L6?

---

I think that is because we don't set `private: true` in RC as we do in eg. accounts
https://github.com/kivra/kivra_accounts/blob/da4076ce8db0d22c8dad66bf99f2469bcbaecc06/package.json#L4

---

So, as long as we say `private: true` for packages we should not publish I think we are fine.

---

Ok, nice. Let's try with your suggestion 👍

---

Let's try it out https://github.com/kivra/kivra_accounts/pull/471